src/services/db/RepoService.ts

```diff
@@ -113,9 +113,36 @@ export default class RepoService extends GitHubService {
   }
 
   async create(
-    sessionData: any,
-    { content, fileName, directoryName, isMedia = false }: any
-  ): Promise<any> {
+    sessionData: UserWithSiteSessionData,
+    {
+      content,
+      fileName,
+      directoryName,
+      isMedia = false,
+    }: {
+      content: string
+      fileName: string
+      directoryName: string
+      isMedia?: boolean
+    }
+  ): Promise<{ sha: string }> {
+    if (this.isRepoWhitelisted(sessionData.siteName)) {
+      logger.info("Writing file to local Git file system")
```

Comment:
can we log for which repoName is this along with the dirName and fileName?

Reply:
https://github.com/isomerpages/isomercms-backend/pull/892/commits/20d3539fadd5f6e1abbe84a363637c69a9172a19